wordpress-mobile/WordPress-iOS
# Dark Mode: Notifications

Refs #12320. This PR implements dark mode fixes for the Notifications section of the app.

![dark-mode-notifications](https://user-images.githubusercontent.com/4780/63629076-e9238280-c607-11e9-8791-9b9c7b5434b0.png)

**To test:**

* Build and run in light and dark mode, and iOS 12
* Check for anything that doesn't look right

Update release notes:


cc @mattmiklic
